app/models/concerns/curation_concerns/nested_works.rb

```diff
@@ -0,0 +1,14 @@
+module CurationConcerns
+  module NestedWorks
+    extend ActiveSupport::Concern
+
+    included do
+      class_attribute :valid_child_concerns
+      self.valid_child_concerns = CurationConcerns::ClassifyConcern.new.all_curation_concern_classes
```

Comment:
I would definitely like to see example config for this included somewhere (in the main curation_concerns initializer?).  Something like:

```
# By default, any work type is a valid child.  To override to limit to certain types, set valid_child_concerns
# Book.valid_child_concerns = [Page]
```